### PR TITLE
📖 amp-subscriptions iframe local platform doc

### PR DESCRIPTION
**Pull request by jpettitt**
Doc for the new `iframe` mode local platform

### CODE REVIEW

File: extensions/amp-access/0.1/iframe-api/README.md [hunk 1]
```diff
@@ -87,14 +116,37 @@ The `config` argument in the `connect` method will contain the original document
 }
 ```
 
+When used with `amp-subscriptions` the `pageProtocol` is set
```
Comment (dvoytenko) [inline extensions/amp-access/0.1/iframe-api/README.md]: Hmm. I don't remember "pageProtocol" in the data structure. Are you sure? Isn't the protocol provided as an `init()` argument?

File: extensions/amp-access/0.1/iframe-api/README.md [hunk 2]
```diff
@@ -87,14 +116,37 @@ The `config` argument in the `connect` method will contain the original document
...
+to `amp-subscriptions` and a `pageConfig` object is included.
+
+```
+{
+  "pageProtocol": "amp-iframe",
```
Comment (dvoytenko) [inline extensions/amp-access/0.1/iframe-api/README.md]: ... but in either case, should it be "amp-subscriptions"?

File: extensions/amp-access/0.1/iframe-api/README.md [hunk 3]
```diff
@@ -87,14 +116,37 @@ The `config` argument in the `connect` method will contain the original document
...
+  "pageConfig": {
+    "productId": "...",
+    "publicationId": "..."
+  },
+  // Remainder as per `amp-access`
+}
+```
+
+The `publicationId` and `productId` are extracted from the
+parent document strucuted data.
```
Comment (dvoytenko) [inline extensions/amp-access/0.1/iframe-api/README.md]: "structured" (unless I'm also misspelling it)

File: extensions/amp-access/0.1/iframe-api/README.md [hunk 4]
```diff
@@ -87,14 +116,37 @@ The `config` argument in the `connect` method will contain the original document
...
+
 
 ## Authorize method
 
 The `authorize` method checks whether the user should be able to access this document. It's expected to be an open-ended JSON structure that can be used for access expressions.
 
 Strong timeout and one-behind semantics are observed for authorization call. If the `authorize()` method does not return within a 3s timeout, the previously returned authorization response is used. If no previous response is available or it's too old, the `defaultResponse` value from the configuration is used. However, even in case of timeout, the iframe authorization will continue until fully complete and will be made available for the next authorization attempt.
 
+In the case of `amp-subscriptions` the authorization response must
+conform to the `local` service authorization struture.
```
Comment (dvoytenko) [inline extensions/amp-access/0.1/iframe-api/README.md]: Ditto "structure". Also maybe worth linking here?

File: extensions/amp-access/0.1/iframe-api/README.md (hunk 1 above)
Comment (jpettitt) [reply extensions/amp-access/0.1/iframe-api/README.md]: oops you are right I was stashing it in pageProtocol in my example, will update.

**Pull request merged**